sdk/spring/azure-spring-boot-starter-metrics/pom.xml

```diff
@@ -58,8 +58,8 @@
             <bannedDependencies>
               <includes>
                 <include>com.microsoft.azure:azure-spring-boot:[2.3.3-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:azure-spring-boot;current} -->
-                <include>io.micrometer:micrometer-core:[1.3.0]</include> <!-- {x-include-update;spring_io.micrometer:micrometer-core;external_dependency} -->
-                <include>io.micrometer:micrometer-registry-azure-monitor:[1.3.0]</include> <!-- {x-include-update;spring_io.micrometer:micrometer-registry-azure-monitor;external_dependency} -->
+                <include>io.micrometer:micrometer-core:[1.3.0]</include> <!-- {x-include-update;io.micrometer:micrometer-core;external_dependency} -->
```

Comment:
Again, io.micrometer:micrometer-core shouldn't need to be in here. Only io.micrometer:micrometer-registry-azure-monitor is a direct dependency

Reply:
Done.